app/controllers/comments_controller.rb

```diff
@@ -1,9 +1,35 @@
 class CommentsController < ApplicationController
+  before_action :set_post, only: %i[new create update]
   def index; end
 
   def show; end
 
-  def new; end
+  def new
+    @comment = Comment.new
+  end
```

Comment:
- [ ] **[OPTIONAL]** On the post show page it would be nice to have a button that takes the user to the new comment form, instead of having to type the URL everytime.